packages/ui/src/2_molecules/TableBase/TableBase.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import { Story } from '@storybook/react';
+
+import React, { ComponentProps } from 'react';
+
+import { prettyTx } from '../../utils';
+import { TableBase } from './TableBase';
+import { Align } from './TableBase.types';
+
+const columns = [
+  {
+    id: 'index',
+    title: 'Index',
+    align: Align.center,
```

Comment:
pls replace all columns with `Align.left`, it should be by default, and need adjustments with the table header, when we change it to the left position, see pics:

Current version:
<img width="545" alt="worktable" src="https://user-images.githubusercontent.com/74987028/197039290-cd8b4890-9835-47b5-9175-933e41d64ec7.png">
Design version:
<img width="347" alt="tableOriginal" src="https://user-images.githubusercontent.com/74987028/197039294-0fde3744-7ba6-48ec-a496-0215f5f925f9.png">